.circleci/config.yml

```diff
@@ -814,19 +829,24 @@ defaults:
       binary_type: native
       image: cimg/node:18.16
 
-  - job_native_test_ext_chainlink: &job_native_test_ext_chainlink
+  - job_native_compile_ext_chainlink: &job_native_compile_ext_chainlink
       <<: *requires_b_ubu_static
-      name: t_native_test_ext_chainlink
+      name: t_native_compile_ext_chainlink
       project: chainlink
       binary_type: native
-      image: cimg/node:18.16
+      # NOTE: test suite disabled due to dependence on a specific version of Hardhat
+      # which does not support shanghai EVM.
+      compile_only: 1
+      image: cimg/node:16.20
```

Comment:
Why downgrade the nodejs version?

Reply:
Because it requires node 16 (fails with a message saying that if your run it on 18).

I think this is at 18 currently only because we were bumping versions last time while this test was disabled and didn't notice that it does not work on it.